### ISSUE TITLE
Breaking changes in the DMA API in 5.18

### ISSUE THREAD

**Issue by ervinpopescu** [tomaspinho/rtl8821ce#269]
### Relevant links and the summary of the errors
* https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=v5.18-rc1&id=63d12cc30574eec9241932cefa17a21df3afd8d0
* https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=v5.18-rc1&id=a2e759612e5ff3858856fe97be5245eecb84e29b

So far, I've managed to get rid of the errors regarding this by manually going in and changing the errors one by one, but there are these seemingly unrelated errors:

```
[...]multiple definition of `is_multicast_mac_addr'[...]

[...]multiple definition of `is_broadcast_mac_addr'[...]

[...]multiple definition of `is_zero_mac_addr'[...]
```

### The complete log:

https://pastebin.com/xDK0yc8A

**Comment by tomaspinho** [comment]
Thanks for reporting this. A fix for the DMA changes is here: https://github.com/tomaspinho/rtl8821ce/pull/275

**Comment by ervinpopescu** [comment]
> Thanks for reporting this. A fix for the DMA changes is here: #275

Still not compiling... It's more or less the same issue as before.

[Log](https://pastebin.com/wfQJAR9t)
